### DISCUSSION TITLE
There are some accessibility issues in the website that need improvement

### DISCUSSION THREAD

**Discussion by redudant-overflowing** [simple-login/app#708] [Feature Requests]
Texts are too small. Also there is lack contrast between the foreground and the background in text and buttons.

# Font size
For example, the text is too small in some places:
<img src="https://user-images.githubusercontent.com/93951203/143771989-f8ea1cb8-464c-4c32-9988-988f544a29dd.png" width="400"/>
<img src="https://user-images.githubusercontent.com/93951203/143772474-ac51737f-7a5f-41f4-88e1-9270d9b98d49.png">

# Color 
The font is grey (#9aa0ac) and the background is white. According to [Contrast Checker](https://webaim.org/resources/contrastchecker/?fcolor=9AA0AC&bcolor=FFFFFF), there is a ratio of 2.62:1 of contrast.
<img src="https://user-images.githubusercontent.com/93951203/143772009-47bbc67a-ae6b-4ee7-8183-ad59490d6d81.png" width="400"/>

People who are color blind can have trouble while seeing this page:
<img src="https://user-images.githubusercontent.com/93951203/143772162-127d24f6-4e0f-4b57-bcaa-c7b532795fa3.png" width="400"/> <img src="https://user-images.githubusercontent.com/93951203/143772173-6a25fc0f-f78c-46c0-adfa-21ce47ca147c.png" width="400"/>

This switch also needs contrast improvement:
<img src="https://user-images.githubusercontent.com/93951203/143772350-07b42e7c-3a41-425c-98f7-77ebedbb862d.png" width="400"/>

**Comment by nguyenkims** [comment]
Thanks for the analysis. We are using a library for UI elements and have a plan to upgrade to its latest version which hopefully handles better the accessibility issues. The library is btw https://preview.tabler.io
